ras_party/controllers/account_controller.py

```diff
@@ -0,0 +1,461 @@
+import uuid
+
+from flask import current_app
+from itsdangerous import SignatureExpired, BadSignature, BadData
+from ras_common_utils.ras_error.ras_error import RasError, RasNotifyError
+from sqlalchemy import orm
+from structlog import get_logger
+
+from ras_party.clients.oauth_client import OauthClient
+from ras_party.controllers.error_decorator import translate_exceptions
+from ras_party.controllers.gov_uk_notify import GovUkNotify
+from ras_party.controllers.queries import query_business_by_party_uuid, query_respondent_by_email, \
+    query_respondent_by_party_uuid
+from ras_party.controllers.validate import Validator, IsUuid, Exists
+from ras_party.models.models import Respondent, RespondentStatus, BusinessRespondent, PendingEnrolment, \
+    Enrolment, EnrolmentStatus
+from ras_party.support.public_website import PublicWebsite
+from ras_party.support.requests_wrapper import Requests
+from ras_party.support.session_decorator import with_db_session
+from ras_party.support.transactional import transactional
+from ras_party.support.util import build_url
+from ras_party.support.verification import decode_email_token
+
+log = get_logger()
+
+NO_RESPONDENT_FOR_PARTY_ID = 'There is no respondent with that party ID '
+EMAIL_VERIFICATION_SENT = 'A new verification email has been sent'
+
+
+@translate_exceptions
+@transactional
```

Comment:
I wonder if it would be simplified if each flask request started a transaction, committed or rollbacked based on success or not?